### ISSUE TITLE
Entity standing in the air and get no damage

### ISSUE THREAD

**Issue by Foorcee** [PaperMC/Paper#1838]
In a world, entities are spawned that can not move and can not be harmed.
If we kill them with / kill they will die but if they go on the activation range they will die again.
The items also move up and down in the affected chunk. When I load the world in the single player, the monsters are gone. If desired, I can still send a video. There is no error in the log.

https://i.imgur.com/djjyrD2.png

World:
[World.zip](https://github.com/PaperMC/Paper/files/2833717/World.zip)

Version:
This server is running Paper version git-Paper-504 (MC: 1.13.2) (Implementing API version 1.13.2-R0.1-SNAPSHOT)

**Comment by electronicboy** [comment]
And on the latest build of paper?
And the rest of the issue template?

**Comment by Foorcee** [comment]
Can confirm for Paper build 521

**Comment by Foorcee** [comment]
Examples

https://i.imgur.com/vjePM0k.gifv
http://screen.forceupdate1.de/SSt4bEaiep.gif

**Comment by Foorcee** [comment]
We have disabled the Async Chunk load.
And we often unload the worlds and recharge them

@aikar

**Comment by aikar** [comment]
This sounds like a problem with your plugins. Until valid reproduce steps are provided without plugins nothing we can do

Why did you disable async? I'm about ready to remove that config

**Comment by Gerrygames** [comment]
Our replay system doesn't write chunks correctly when async chunk loading is enabled. Chunks are written when the ChunkLoadEvent is called and when the replay is started all loaded chunks (all chunks returned by ``nmsWorld.getChunkProvider().a()``) are written. However this is probably a pretty special case and won't cause any issues on other servers.

**Comment by Gerrygames** [comment]
We switched back to Spigot on our Skyblock servers. But the issue starts to appear on our other servers too. The servers are running on git-Paper-523. 
As this is marked as incomplete, here's a list of our plugins:
https://puu.sh/CN9FX/fd645ebd2d.png
And another video of the issue on our other server:
https://youtu.be/fEamalPm6mQ
If there's any other information we can provide, just tell us what you need.
Unfortunately we will have to switch back to Spigot if this issue can't be fixed, but Paper really helps improving our performance.

**Comment by electronicboy** [comment]
can you replicate this with a minimal plugin? what are you doing in ChunkLoadEvent?

**Comment by Gerrygames** [comment]
Still present with the following plugins:
https://puu.sh/CNbgl/1035b84cf0.png
Cloud, EndermanEssentials, EndermanNetwork, EndermanServer, EnderStats and LibLoader are not accessing the world, messing with packets or anything similar. They basically only use the Bukkit API to load and register commands.
ProtocolLib shouldn't be the issue.
CrystalSystem is spawning particles effects and pets.
The ReallifeEnd does minimal changes to the world (spawning some mobs and setting a few blocks) in the ChunkLoadEvent. I can provide source code, if needed.

**Comment by Gerrygames** [comment]
Just tested with only LibLoader and Cloud. Both plugins should not have any impact on this. LibLoader is only loading classes from jars into the classpath and Cloud is only registering the server in our backend software. The issue still persists. If we enable asny chunk loading the issues disappears.

**Comment by zachbr** [comment]
Needs to be narrowed down to a plugin (or none) and if that plugin is one of your private ones we would ask you dig a bit and figure out how its occurring, we are not going to be able to do a lot of guessing with private plugins.

**Comment by Proximyst** [comment]
As we've been waiting on reporter for 1y+ now, I'll close this. Open a new issue if this issue is encountered again.
